cypress/integration/cards_spec.js

```diff
@@ -1,32 +1,66 @@
+import { aliasPostsQuery } from '../utils/graphql-test-utils';
+
 describe('All posts view', () => {
   beforeEach(() => {
+    cy.intercept(
+      'POST',
+      'https://hivemind-staging-branch.herokuapp.com/graphql',
+      req => {
+        aliasPostsQuery(req, 'posts');
+      }
+    );
+
     cy.visit('http://localhost:3000/');
   });
 
-  it('When you visit the page, you should see posts', () => {});
+  it('When you visit the page, you should see posts', () => {
+    cy.get('.cards-container').children().should('have.length', 3);
+  });
 
-  it('If there are no posts, you should see a prompt', () => {});
+  it('Each post should show who posted it (avatar and username)', () => {
+    cy.contains('Posted by BeeKeeper1');
```

Comment:
shouldn't you grab the specific card first? Or if it's passing does it not matter?

Reply:
I can change it to make it more specific if you'd like! I think it's just a question of how specific we want to be with the test